core/src/main/resources/log4j4ibis.properties

```diff
@@ -1,15 +1,15 @@
 
 # maximum file size of plain log files, can also be KB, e.g. 512KB
-log.maxFileSize=1MB
+log.maxFileSize=10MB
 # maximum number of history logfiles
 log.maxBackupIndex=9
 
 # maximum file size of xml log files
-log.xml.maxFileSize=1MB
+log.xml.maxFileSize=10MB
 log.xml.maxBackupIndex=9
 
 # maximum file size of msg log files
-log.msg.maxFileSize=1MB
+log.msg.maxFileSize=10MB
 log.msg.maxBackupIndex=9
```

Comment:
de xml size mag weg, en ik denk toch dat het beter is om hem op 1MB te laten.
In de FrankLogConfigurationFactory#setLevel kan de default naar INFO